src/Moesif/Middleware/MoesifLaravel.php

```diff
@@ -268,19 +268,21 @@ public function handle($request, Closure $next)
 
         $responseData = [
             'time' => $endDateTime->format('Y-m-d\TH:i:s.uP'),
-            'status' => $response->status()
+            'status' => $response->getStatusCode()
         ];
 
 
-        $responseContent = $response->content();
+        $responseContent = $response->getContent();
         if ($logBody && !is_null($responseContent)) {
-          $jsonBody = json_decode($response->content(), true);
+            $jsonBody = json_decode($response->getContent(), true);
 
           if(!is_null($jsonBody)) {
               if (!is_null($maskResponseBody)) {
                   $responseData['body'] = $maskResponseBody($jsonBody);
+                  $responseData['transfer_encoding'] = 'json';
```

Comment:
```
if (cond) {
  ...
  foo();
} else {
  ...
  foo();
}
```

```
if (cond) {
  ...
}
foo();
```

Reply:
Added encoding to ensure the type of body we are sending, we were doing it for base64 encoded body. Other changes have been made to use the base method function that could be accessible by all Response object. For example - Response content could be fetched by  `$response->getContent();` or  `$response->content();` but some response object won't have `$response->content();` which would result into error, all response object will have base class method `$response->getContent();`. More info here -  https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Response.php#L397